sdk/core/loader/boot.cc

```diff
@@ -105,13 +105,18 @@ namespace
 		 */
 		FirstDataSealingType = 9,
 
+		/**
+		 * The sealing type used for sealed export table entries.
```

Comment:
Why does the comment say export table entries but the name is `ImportTableEntries`?